cirq-google/cirq_google/serialization/circuit_serializer.py

```diff
@@ -53,7 +59,10 @@ class CircuitSerializer(serializer.Serializer):
     """
 
     def __init__(
-        self, USE_CONSTANTS_TABLE_FOR_MOMENTS=False, USE_CONSTANTS_TABLE_FOR_OPERATIONS=False
+        self,
+        USE_CONSTANTS_TABLE_FOR_MOMENTS=False,
+        USE_CONSTANTS_TABLE_FOR_OPERATIONS=False,
+        default_package_name='cirq_google',
```

Comment:
Unused argument - please add a comment if that is intentional.

Reply:
Removed.